bears/java/PMDBear.py

```diff
@@ -0,0 +1,31 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "pmd -R java-basic," + \
+                "java-braces," + \
+                "java-clone," + \
+                "java-codesize," + \
+                "java-controversial," + \
+                "java-design," + \
+                "java-imports," + \
+                "java-naming," + \
+                "java-optimizations," + \
+                "java-strings," + \
+                "java-unnecessary," + \
+                "java-unusedcode" + \
```

Comment:
please file at least an issue so the parameters for each of those options are exposed to the user